superflore/generators/ebuild/gen_packages.py

```diff
@@ -42,7 +42,10 @@
 org_license = "BSD"
 
 
-def regenerate_pkg(overlay, pkg, distro, preserve_existing=False):
+def regenerate_pkg(overlay, pkg, distro, preserve_existing, skip_keys):
```

Comment:
I don't remember what the implication of `preserve_existing` not defaulting to `False` is, but would probably feel better if it continued to default to `False`.

Reply:
No worries; done.